### PR TITLE
Refactor start-ec CMD to multiline format

### PR DESCRIPTION
**Pull request by naps62**
This should greatly increase readability and also improve diffing for future changes to these flags

### CODE REVIEW

File: install/scripts/start-ec.sh [hunk 1]
```diff
@@ -123,7 +139,27 @@ if [ "$CLIENT" = "nethermind" ]; then
...
+      --JsonRpc.Port ${EC_HTTP_PORT:-8545} \
+      --JsonRpc.EnginePort ${EC_ENGINE_PORT:-8551} \
+      --JsonRpc.EngineHost 0.0.0.0 \
+      --Sync.AncientBodiesBarrier 1 \
+      --Sync.AncientReceiptsBarrier 1 \
+      --Sync.SnapSync true \
+      --Merge.Enabled true \
+      --JsonRpc.JwtSecretFile=/secrets/jwtsecret \
+      $EC_ADDITIONAL_FLAGS"
+||||||| 99a3e21
```
Comment (jclapis) [inline install/scripts/start-ec.sh:158]: Looks like you were in the middle of a merge conflict when you pushed this, can you resolve it?

File: install/scripts/start-ec.sh [hunk 2]
```diff
@@ -172,12 +208,44 @@ if [ "$CLIENT" = "besu" ]; then
...
+      --fast-sync-min-peers=3 \
+      --nat-method=docker \
+      --p2p-host=$EXTERNAL_IP \
+      --engine-rpc-enabled \
+      --engine-rpc-port=${EC_ENGINE_PORT:-8551} \
+      --engine-host-allowlist=* \
+      --engine-jwt-secret=/secrets/jwtsecret \
+      --Xbonsai-use-snapshots=true \
+      $EC_ADDITIONAL_FLAGS"
+||||||| 99a3e21
```
Comment (jclapis) [inline install/scripts/start-ec.sh:239]: Same here

File: install/scripts/start-ec.sh (hunk 1 above)
Comment (naps62) [reply install/scripts/start-ec.sh]: Missed this one. fixed

File: install/scripts/start-ec.sh (hunk 2 above)
Comment (naps62) [reply install/scripts/start-ec.sh:232]: fixed

**Comment by naps62** [comment]
done!

**Comment by jclapis** [comment]
Looks good, thanks!

**Pull request merged**